lib/common.ml

```diff
@@ -40,32 +40,3 @@ module Re2 = struct
   let wrap s = create_exn s
   let unwrap = Re2.to_string
 end
-
-open Devkit
-
-let fmt_error ?exn fmt =
-  Printf.ksprintf
-    (fun s ->
-      match exn with
-      | Some exn -> Error (s ^ " :  exn " ^ Exn.str exn)
-      | None -> Error s)
-    fmt
-
-let first_line s =
-  match String.split_on_char '\n' s with
-  | x :: _ -> x
-  | [] -> s
-
-let decode_string_pad s = Stre.rstrip ~chars:"= \n\r\t" s |> Base64.decode_exn ~pad:false
-
-let http_request ?headers ?body meth path =
-  let setup h =
-    Curl.set_followlocation h true;
-    Curl.set_maxredirs h 1
-  in
-  match%lwt Web.http_request_lwt ~setup ~ua:"monorobot" ~verbose:true ?headers ?body meth path with
-  | `Ok s -> Lwt.return @@ Ok s
-  | `Error e -> Lwt.return @@ Error e
-
-let sign_string_sha256 ~key ~basestring =
-  Cstruct.of_string basestring |> Nocrypto.Hash.SHA256.hmac ~key:(Cstruct.of_string key) |> Hex.of_cstruct |> Hex.show
```

Comment:
I've moved these more general/utility functions to the `util.ml` file as I wanted to add a function to this file, but it would cause cyclic dependencies. And these functions actually belong more in a util file than here, so I took the opportunity to move them